include/robot_localization/navsat_conversions.hpp

```diff
@@ -253,7 +257,9 @@ static inline void LLtoUTM(
 
   // Compute the UTM Zone from the latitude and longitude
   char zone_buf[] = {0, 0, 0, 0};
-  snprintf(zone_buf, sizeof(zone_buf), "%d%c", ZoneNumber,
+  // We &0x3fU to let GCC know the size of ZoneNumber. In this case, it's under
+  // 63 (6bits)
+  snprintf(zone_buf, sizeof(zone_buf), "%d%c", ZoneNumber & 0x3fU,
```

Comment:
Again, the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Streams) says to use streams where appropriate (I would say they are appropriate here) instead of the C equivalents. See question above about where to submit this change.

Reply:
Similar comment as above: If there's other situations where we do this, I don't think its worth changing at this stage